pylzy/lzy/api/v1/__init__.py

```diff
@@ -61,11 +61,18 @@ def op(
     env: Env = Env(),
     description: str = "",
     lazy_arguments: bool = False,
-    env_variables: Optional[Mapping[str, str]] = None
+    env_variables: Optional[Mapping[str, str]] = None,
+    docker_only: bool = False
 ):
     if env_variables is None:
         env_variables = {}
 
+    if docker_only and (conda_yaml_path is not None
+                        or local_modules_path is not None
+                        or libraries is not None
+                        or python_version is not None):
+        raise ValueError("docker_only set, but some other env property requires sharing local env")
```

Comment:
I'd suggest the next error message -- `'docker_only' is not compatible with explicit python env settings (conda_yaml_path, local_modules_path, libraries, python_version)`

Let's get @mrMakaronka to help

Reply:
Agree with @imakunin